packages/twenty-front/src/modules/object-record/record-field/form-types/components/VariableChip.tsx

```diff
@@ -90,19 +94,28 @@ export const VariableChip = ({
       isFullRecord,
     });
 
-  const label = isDefined(variableLabel)
-    ? variableLabel
-    : extractRawVariableNamePart({
-        rawVariableName,
-        part: 'selectedField',
-      });
+  const isVariableNotFound = !isDefined(variableLabel);
+  const label = isVariableNotFound ? 'Not Found' : variableLabel;
```

Comment:
I think we should progressively add internationalisation to the workflows. Could we use `const { t } = useLingui()` and call ```t`Not Found` ```?